vscode.md

```diff
@@ -0,0 +1,53 @@
+# express-server
+
+ # Features of VS Code
+
+1.**A very helpful syntax highlighter and autocomplete feature that provides smart completions based on variable types, function definitions, and imported modules.**
+
+2.**The built in debugger helps you accelerate editing, compiling, and debugging by adding breakpoints and watchers. By default it comes with support for NodeJS and can debug anything that is transpiled to JavaScript but other runtimes like C++ or Python will require an extension to be installed.**
+
+3.**There is a built in GUI for Git for the most common commands which makes it really easy to instantly see the changes you're making in your project.**
+
+4.**VS Code offers icon theming so that you can switch between icon themes in the same manner you'd switch code themes.**
+
+5.**The ability to work with a full featured integrated terminal let's you run everything you normally would from your terminal of choice.** 
```

Comment:
Unbold this, only headings must be in bold rest can be in normal font weight

Reply:
Done